### ISSUE TITLE
Do we need the render texture approach anymore?

### ISSUE THREAD

**Issue by TerryCavanagh** [TerryCavanagh/haxegon#110]
It's a significant redesign, but Starling's viewport stuff might make this obsolete - we can have pixel perfect scaling without needing a separate canvas.

What this probably comes down to is simply: is it worth the work? Do some trials and benchmarks and see if it even makes a difference; if it doesn't, no big deal, no need to change anything. If it does...

**Comment by TerryCavanagh** [comment]
see http://forum.starling-framework.org/topic/blitting-using-quadbatch-is-slow-what-am-i-doing-wrong

**Comment by TerryCavanagh** [comment]
Been investigating this a bit.

From what I've tried, it's not clear that trying to keep a stage with a display list instead of a render texture will actually lead to any speed increases. Obviously if I changed Haxegon to just fall back and use a display list approach instead of it's fake "immediate mode" then we'd get some speed increases, but that would be a pretty huge restructure of the engine and defeats the point of if a little bit (why not just use Starling, in that case?).

Gonna poke at if for another bit, but probably giving up on this approach.

If any one reading has any suggestions about this, now is a good time to chime in!
